backend/api/models.py

```diff
@@ -1,3 +1,24 @@
+from django.conf import settings
 from django.db import models
 
 # Create your models here.
+from django.dispatch import receiver
+from django_rest_passwordreset.signals import reset_password_token_created
+from django.core.mail import send_mail
+
+from templates.consts import *  
+
+
+@receiver(reset_password_token_created)
+def password_reset_token_created(sender, instance, reset_password_token, *args, **kwargs):
+
+    base_url = 'https://sharif-wss.ir/'  # TODO
+    email_message = "{}?token={}".format(base_url, reset_password_token.key)
```

Comment:
I think the `base_url` should be passed to the `reset_password` API as a callback URL, like the payment procedure. It's not good for this to be hardcoded.

Reply:
I agree. I'll see if I can override its view

Reply:
Will be discussed with the front team later